W-13236494-AWS-PS (#127)
mulesoft/docs-hosting: cloudhub-2/modules/ROOT/pages/ps-config-advanced.adoc

[[configure-aws-role]]
== Configure AWS Service Role

> If you have identity and access management (IAM) roles configured in AWS, you can associate the role with your private space. The private space receives the permissions from the IAM role in AWS and can access AWS resources. To enable this feature, Anypoint Platform generates a unique role name you can use with AWS IAM. To configure this feature, you must use the same organization ID as the organization that originally configured the private space. You must also allow outbound traffic on port 443 to use this feature. For more information, see https://docs.aws.amazon.com/IAM/latest/UserGuide/id_roles.html[IAM roles^]. 

Reviewer: configured in/for the ps? 
The organization didn't configure the ps, right?

Author: ```suggestion
If you have identity and access management (IAM) roles configured in AWS, you can associate the role with your private space. The private space receives the permissions from the IAM role in AWS and can access AWS resources. To enable this feature, Anypoint Platform generates a unique role name you can use with AWS IAM. To configure this feature, you must use the same organization ID as the organization in which private space was originally configured. You must also allow outbound traffic on port 443 to use this feature. For more information, see https://docs.aws.amazon.com/IAM/latest/UserGuide/id_roles.html[IAM roles^]. 
```

Reviewer: It's a little confusing where you configure what.

Something like:

To configure this feature in AWS:
- Use the unique role that Anypoint Platform generates.
- Use the organization ID originally configured for the private space.
- Allow inbound traffic on port 443 for the private space. (?)

Reviewer: should it be "the private space"?

Reviewer: Read this paragraph out loud. Repeated "use/configure this feature". Is there a way to tighten and clarify?

Author: ```suggestion
If you have identity and access management (IAM) roles configured in AWS, you can associate the role with your private space. The private space receives the permissions from the IAM role in AWS and can access AWS resources. To configure this feature in AWS: 

* Use the unique AWS IAM role name that Anypoint Platform generates.
* Use the organization ID for the organization in which the private space was configured. 
* Allow outbound traffic on port 443 to use this feature. 

For more information, see https://docs.aws.amazon.com/IAM/latest/UserGuide/id_roles.html[IAM roles^]. 
```
